lib/masdb/initializer.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Masdb.Initializer do
+  def start_link do
+    {:ok, spawn_link(__MODULE__, :initialize_node, [])}
+  end
+
+  def initialize_node do
+    nodes = Masdb.Node.list()
+    {:ok, answers} = Masdb.Node.DistantSupervisor.query_remote_nodes_until(
```

Comment:
I have to say, I like your usage of "æ". It implecitely already does that.

It uses [`query_remote_nodes`](https://github.com/DrPandemic/MasDB/blob/master/lib/masdb/distant_supervisor.ex#L14) which already has a maximum timeout per query. This can be modified (https://github.com/DrPandemic/MasDB/blob/54f0f60c4935417c1ec800a1fb4aeebf5e10a024/config/config.exs#L15).

So, this is kinda answering your question. We have a timeout per query, but not for the complete set. Do you think it's enough?